custom-apps/better-local-files/package.json

```diff
@@ -15,11 +15,6 @@
     "license": "MIT",
     "devDependencies": {
         "@types/pixelmatch": "^5.2.4",
-        "@types/react": "^18.0.28",
-        "@types/react-dom": "^18.0.11",
-        "spicetify-creator": "^1.0.13"
-    },
-    "dependencies": {
         "lucide-react": "^0.115.0",
```

Comment:
Shouldn't these stay in `dependencies` instead of `devDependencies`?

Reply:
no, dependencies are packages needed to **run** a project